File: Aula_1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void){
    int i;

    do{
        i = get_int ("Altura: \n");
    }
    while (i < 1 || i > 8);



    for (int x = 0; x < i; x++)
    {

        for (int y1 = 0; y1 <= i-x-2; y1++){
        printf(" ");
        }
        for (int y = 0; y <= x; y++){

        printf("#");
        }
    printf("\n");
    }

}

File: Aula_6/Mario.py
from cs50 import get_int

def main():
    x = altura_torre()
    for i in range(0, x, 1):
        for j in range(1, x-i , 1):
            print(" ", end = "")
        for k in range(0, i+1 , 1):
            print("#", end = "")
        print()



def altura_torre():
    while True:
        height = get_int("Height: ")
        if (height > 0 and height < 9):
            break
    return height

main()